storage/libradb/src/lib.rs

```diff
@@ -898,7 +898,7 @@ impl LibraDBTrait for LibraDB {
             .ledger_info()
             .version();
         let events = self
-            .get_events_by_event_key(event_key, start, false, limit, version)?
+            .get_events_by_event_key(event_key, start, true, limit, version)?
```

Comment:
this breaks how we get the latest events from this endpoint (start == u64::max_value) (which breaks the leader reputation algorithm)

Reply:
@zekun000: can you refer to call site? Why it uses LibraDBTrait at all?